master/reference/calicoctl/commands/label.md

```diff
@@ -0,0 +1,153 @@
+---
+title: calicoctl label
+canonical_url: 'https://docs.projectcalico.org/v3.5/reference/calicoctl/commands/apply'
+---
+
+This sections describes the `calicoctl label` command.
```

Comment:
This is a case of subject verb disagreement. So, you can just do `s/sections/section`